.claude-skills/registry/SKILL_REGISTRY.md

```
@@ -169,7 +169,7 @@ Products:
 | circle | ✓ | ✓ | ✓ |
 | legacy | ✓ | ✓ | ✓ |
 
-### Technical Skills (19)
+### Technical Skills (10)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)